openhcl/underhill_attestation/src/igvm_attest/ak_cert.rs

```diff
@@ -12,8 +12,8 @@ use zerocopy::FromBytes;
 /// AkCertError is returned by parse_ak_cert_response() in emuplat/tpm.rs
 #[derive(Debug, Error)]
 pub enum AkCertError {
-    #[error("AK cert response size is too small to parse")]
-    SizeTooSmall,
+    #[error("AK cert response is too small to parse. Found {size} bytes but expected at least {expected_size}")]
+    SizeTooSmall { size: usize, expected_size: usize },
```

Comment:
Would `minimum_size` be more precise?

Reply:
Yeah, it would be. Updated

Reply:
I was going to use `found` instead of `size` too because I felt like that was clearer, but I instead kept the field names inline with the other `AkCertError`s